test/libponyc/refer.cc

```diff
@@ -838,3 +838,14 @@ TEST_F(ReferTest, ConsumeParamVarSubfieldReassignSameExpressionReuse)
   TEST_ERRORS_1(src,
     "can't use a consumed local or field in an expression");
 }
+
+TEST_F(ReferTest, ConsumeTupleAccessorOfFunctionValResult)
+{
+    const char* src = "actor Main\n"
+                      "    new create(env: Env) =>\n"
+                      "       let a = \"I am a string\"\n"
+                      "       consume a.chop(1)._1\n";
```

Comment:
It's a small nitpick, but for consistency reasons, please use the same format for test case source code that the other test cases use:
```suggestion
    const char* src =
      "actor Main\n"
        "new create(env: Env) =>\n"
          "let a = \"I am a string\"\n"
          "consume a.chop(1)._1";
```